cpp/src/parquet/arrow/reader_writer_benchmark.cc

```diff
@@ -95,15 +97,37 @@ void SetBytesProcessed(::benchmark::State& state) {
   state.SetBytesProcessed(bytes_processed);
 }
 
+constexpr int64_t kAlternatingOrNa = -1;
+
+template <typename T>
+std::vector<T> RandomVector(int64_t true_percentage, int64_t vector_size,
```

Comment:
Can't this be factored out in `testing/random.h`?

Reply:
It'll need to depend on libarrow_testing.so, not sure if this is a problem.

Reply:
If it is OK, I'd prefer to save this refactoring for a later point in time, in case more changes are need to this.